doc/netplan.md

```diff
@@ -277,6 +277,11 @@ Virtual devices
     Example to enable all link-local addresses: ``link-local: [ ipv4, ipv6 ]``
     Example to disable all link-local addresses: ``link-local: [ ]``
 
+``configure-without-carrier`` (bool)
```

Comment:
my preference here is for the 'ignore-carrier' syntax, I think 'configure-without-carrier' is unnecessarily verbose.